WordPressVIPMinimum/Sniffs/Variables/VariableAnalysisSniff.php

```diff
@@ -728,8 +728,7 @@ protected function checkForFunctionPrototype(
 				// We haven't been defined by this point.
 				$phpcsFile->addWarning( "Variable `%s` is undefined.", $stackPtr,
 					'UndefinedVariable',
-					array( "\${$varName}" ),
-					'WordPressVIPMinimum.Variables.VariableAnalysis'
+					array( "\${$varName}" )
```

Comment:
These changes were already merged via a different PR. Please pull `master` into this PR's branch, so that we don't re-commit the changes and possibly lose the commit message and reference from the first effort.